cachito/workers/scm.py

```diff
@@ -209,6 +209,10 @@ def fetch_source(self, gitsubmodule=False):
                 OSError,  # raised by tarfile when an FS operation fails
             ) as exc:
                 log.warning("Error handling archived artifact '%s': %s", previous_archive, exc)
+                log.info(
```

Comment:
Should this also be a `log.warning` ?

Reply:
Not sure, since the idea here is to announce Cachito is recovering from the issue raised in the previous warning...

Reply:
IMO `log.info` is fine